### ISSUE TITLE
How to  configure weekly / scheduled release cycles?

### ISSUE THREAD

**Issue by kelvindecosta** [google-github-actions/release-please-action#670]
### TL;DR

I've tried to setup `release-please` such that:
- every week (github action scheduled via CRON) `release-please` submits a PR for a candidate release
- once that PR is merged, `release-please` creates releases on GitHub releases

I'm currently using two actions:
- `release-pr.yml` -> submits PR for candidate release weekly, using the `manifest-pr` command
- `github-release.yml` -> on push to `main`, runs the `manifest` command

### Expected behavior

The schedule triggers a new PR for a release candidate.
After merging this PR, the action then creates GitHub releases.

### Observed behavior

The `manifest` command, in the action that runs on all pushes to `main`, always creates a candidate PR on every push to `main`.

### Action YAML

```yaml
on:
  push:
    branches:
      - main

name: GitHub Release
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          command: manifest

---

on:
  workflow_dispatch:
  schedule:
    - cron: "0 0 * * SAT"

name: Propose Release Candidate via PR
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          command: manifest-pr
```

### Log output

```text
NA
```

### Additional information

`.release-please-manifest.json`
```json
{
  ".": "<our-project-version>" 
}
```

`release-please-config.json`
```json
{
  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
  "group-pull-request-title-pattern": "chore: release ${version}",
  "packages": {
    ".": {
      "changelog-path": "CHANGELOG.md",
      "release-type": "python",
      "bump-minor-pre-major": false,
      "bump-patch-for-minor-pre-major": false,
      "draft": false,
      "prerelease": false,
      "extra-files": [
        {
          "path": "package.json",
          "type": "json",
          "jsonpath": "$.version"
        }
      ]
    }
  }
}
```
